packages/compass-web/polyfills/@mongodb-js/saslprep/index.ts

```diff
@@ -0,0 +1,7 @@
+export function saslprep(): void {
+  // no-op
+}
+
+export default {
```

Comment:
the named export above is the only one the driver uses. 

But I understand wanting to cover your bases by making the "compatible" with the actual saslprep. If we want that then we'd need the `export =` as well.


No strong feelings though